sklearn/covariance/_shrunk_covariance.py

```diff
@@ -534,6 +534,27 @@ class OAS(EmpiricalCovariance):
       coefficient in the convex combination used for the computation
       of the shrunk estimate. Range is [0, 1].
 
+    Examples
+    --------
+    >>> import numpy as np
+    >>> from sklearn.covariance import OAS
+    >>> from sklearn.datasets import make_gaussian_quantiles
+    >>> real_cov = np.array([[.8, .3],
+    ...                      [.3, .4]])
+    >>> rng = np.random.RandomState(0)
+    >>> X = rng.multivariate_normal(mean=[0, 0],
+    ...                                    cov=real_cov,
+    ...                                    size=500)
+    >>> oas = OAS().fit(X)
+    >>> oas.covariance_
+    array([[0.7533..., 0.2763...],
+        [0.2763..., 0.3964...]])
```

Comment:
I think adding a few spaces here to align the brackets with the previous line would make it look nicer.

Reply:
I agree. Fixed it.